### PR TITLE
Add multiple contents extension specification

### PR DESCRIPTION
**Pull request by javagl**
This adds the multiple contents extension specification that was developed in the 3d-tiles-next branch, based on the latest state after https://github.com/CesiumGS/3d-tiles/pull/517

Direct link to README.md for preview: https://github.com/javagl/3d-tiles/blob/multiple-contents-to-main/extensions/3DTILES_multiple_contents/README.md

### CODE REVIEW

File: extensions/3DTILES_multiple_contents/README.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
+<!-- omit in toc -->
+# 3DTILES_multiple_contents
+
+<!-- omit in toc -->
+## Contributors
+
+* Sean Lilley, Cesium
+* Peter Gagliardi, Cesium
+* Sam Suhag, Cesium
```
Comment (ptrgags) [inline extensions/3DTILES_multiple_contents/README.md:9]: Let's update the contributor's list

```suggestion
* Sam Suhag, Cesium
* Marco Hutter, Cesium
* Don McCurdy, Independent
```

File: extensions/3DTILES_multiple_contents/README.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+        "subtrees": {
+          "uri": "subtrees/{level}/{x}/{y}.subtree"
+        }
+      }
+    }
+  }
+}
+```
+
+## Extension Schema Reference
```
Comment (ptrgags) [inline extensions/3DTILES_multiple_contents/README.md:339]: For consistency with the other specs, remove the generated schema reference and just link to the schema files

**Review by ptrgags — COMMENTED** [review]
@javagl did a review pass, just a couple things that need updating.

**Comment by javagl** [comment]
@ptrgags The schema reference has been removed. 

For consistency, I also considered to use the style for the examples that we recently seemed to settle to...

> **Example:** What this is

But I am/was not sure whether such changes are still desired in the staging branch, and ... essentially, the whole README would then consist of not much more than "headlines + Examples" ....

**Comment by ptrgags** [comment]
@javagl I agree, the 

>**Example**

blocks would look distracting in this specification since it's relatively simple. 

The changes look good, @lilleyse did you want to take a look before we merge?

**Comment by lilleyse** [comment]
I have a few larger changes I'd like to make but will open a new PR for those

**Pull request merged**